Java Script/ex-22/index.js

```diff
@@ -0,0 +1,12 @@
+"use strict"
+alert("Enter two numbers to see if the first is greater than the second");
+
+let message = Number(prompt(firstNumber));
+let message = Number(prompt(secondNumber));
```

Comment:
If you run this code, it will give you error. Please run it and check which mistake it is
And fixt it. 😉